bazel/deps.bzl

```diff
@@ -0,0 +1,15 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def platformio_rules_dependencies():
```

Comment:
Would you mind documenting these? Ideally one top level comment in each of the .bzl libraries summarizing what it contains and one liner on each of the defined macros.

That should help any future editors.

Reply:
Done :D